packages/chatbot-server-mongodb-public/src/conversations.eval.ts

```diff
@@ -9,7 +9,11 @@ import {
 import fs from "fs";
 import path from "path";
 import { makeConversationEval } from "./eval/ConversationEval";
+import { getConversationEvalCasesFromBraintrust } from "../../mongodb-rag-core/src/eval/getConversationEvalCasesFromBraintrust";
```

Comment:
```suggestion
import { getConversationEvalCasesFromBraintrust } from "mongodb-rag-core";
```

Rather than using a relative file path import can you update `mongodb-rag-core` index file(s) to export `getConversationEvalCasesFromBraintrust` and then import it using packge import syntax here?